src/models/Choice.model.js

```diff
@@ -0,0 +1,13 @@
+const Sequelize = require('sequelize-views-support');
+const sequelize = require('./sequelize.js');
+
+const Question = require('./Question.model');
+
+const Choice = sequelize.define('choices', {
+  text: { type: Sequelize.STRING(255), allowNull: false },
```

Comment:
it's better to use this `Sequelize.TEXT` or `Sequelize.STRING` instead of `Sequelize.STRING(255)`